data/json/construction.json

```diff
@@ -2279,5 +2279,25 @@
         "time" : 0,
         "pre_special" : "check_no_trap",
         "post_special" : "done_mark_firewood"
+    },{
+        "type" : "construction",
+        "description" : "Build Butchering Rack",
+        "category" : "FURN",
+        "required_skills" : [ [ "fabrication", 1 ], [ "survival", 3 ] ],
+        "time" : 90,
+        "qualities": [
+            [ { "id": "DIG", "level": 1 } ],
+            [ { "id": "HAMMER", "level": 2 } ],
+            [ { "id": "CUT", "level": 1 } ],
+            [ { "id": "SAW_W", "level": 1 } ]
+        ],
+        "components" : [
+            [ [ "stick_long", 6 ], [ "2x4", 6 ] ],
+            [ ["string_36", 8 ], [ "rope_makeshift_6", 2 ], [ "rope_6", 2 ], [ "wire", 8 ], [ "vine_30", 1 ] ],
```

Comment:
Ropes could be replaced with `rope_natural` or `rope_natural_short` requirements.